components/Activity/Agenda/FileList.tsx

```diff
@@ -0,0 +1,25 @@
+import { TableCellAttachment } from 'mobx-lark';
+import { FilePreview } from 'mobx-restful-table';
+import { FC } from 'react';
+import { Badge } from 'react-bootstrap';
+
+import { i18n } from '../../../models/Base/Translation';
+
+const { t } = i18n;
+
+export const FileList: FC<{ data: TableCellAttachment[] }> = ({ data }) => (
+  <section>
+    <h2>{t('file_download')}</h2>
+    <ol>
+      {data.map(({ id, name, mimeType, attachmentToken }) => (
+        <li key={id + ''}>
+          <FilePreview
+            type={mimeType}
+            path={`/api/lark/file/${attachmentToken}`}
+          />
+          <Badge>{name}</Badge>
+        </li>
+      ))}
+    </ol>
+  </section>
+);
```

Comment:
```suggestion
export const FileList: FC<{ data: TableCellAttachment[] }> = observer (({ data }) => (
  <section>
    <h2>{t('file_download')}</h2>
    <ol>
      {data.map(({ id, name, mimeType, attachmentToken }) => (
        <li key={id + ''}>
          <FilePreview
            type={mimeType}
            path={`/api/lark/file/${attachmentToken}`}
          />
          <Badge>{name}</Badge>
        </li>
      ))}
    </ol>
  </section>
));
```
用了 MobX-i18n 提供的可观察数据的组件，也要变成观察者。